### DISCUSSION TITLE
Navigation buttons on Top Rated and Reviews

### DISCUSSION THREAD

**Discussion by benbarden** [benbarden/switchscores#87] [General]
Top Rated now has buttons across the top of each page to help with moving between pages in that section.
https://www.switchscores.com/top-rated

This is partly because a lot of people land on the All-time page, and they might find the other pages useful.

Reviews has those buttons too:
https://www.switchscores.com/reviews

I need to add "Reviews home" to those links, but otherwise it's the same.

Anywhere else these might be useful?

**Comment by cosmo0** [comment]
https://www.switchscores.com/games : the "browse switch games" on the right is just not visible enough.

https://www.switchscores.com/games/by-category/adventure : the sub-categories are not visible enough either

**Comment by benbarden** [comment]
Thanks! Games now has the buttons on every page and sub-page. I've also included Recent and Upcoming.

I thought the subcategory links were ok - these buttons wouldn't make that much difference. They would probably look weird on any category that has several subcategories with longer names.
